src/herbivoresAndCarnivores.js

```diff
@@ -1,17 +1,40 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
```

Comment:
```suggestion
  constructor(name, health = 100) {
```
There is no correlation now with the health of Herbivore or Carnivore